src/styles/fonts.css

```diff
@@ -0,0 +1,11 @@
+@font-face {
+  font-family: Avenir;
+  src: url('../fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('../fonts/Avenir-Heavy.ttf');
+  font-weight: 400;
+}
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. It's a good practice to specify a generic family as a fallback, such as 'sans-serif' or 'serif'.